app/move/steps/create.js

```diff
@@ -55,7 +55,14 @@ const riskStep = {
     {
       field: 'from_location_type',
       value: 'prison',
-      next: 'special-vehicle',
+      next: [
+        {
+          field: 'to_location_type',
+          value: 'prison',
+          next: 'save',
+        },
+        'health-information',
```

Comment:
This should remain `special-vehicle`, otherwise it will change the behaviour of the existing prison to court journey.

Reply:
Well spotted.